ep/build-velox/src/build_velox.sh

```diff
@@ -10,8 +10,8 @@ BUILD_PROTOBUF=ON
 BUILD_TYPE=release
 VELOX_HOME=
 #for ep cache
-VELOX_REPO=https://github.com/oap-project/velox.git
-VELOX_BRANCH=main
+VELOX_REPO=https://github.com/zhejiangxiaomai/velox.git
```

Comment:
why use your repo instead of oap-project

Reply:
Just use it for internal jenkens tpcds test. I will back out it when I merge this PR.